webapp/src/main/webapp/resources/template.soy

```diff
@@ -103,4 +103,34 @@
   </table>
   <p>Please fix the parent location type(s) corresponding to the stop location type(s)!</p>
   <br><br>
+{/template}
+
+
+/**
+ * Renders a basic explanation of the validation notice.
+ * @param notices : List<[shapeId : string, csvRowNumber : number]>
+ */
+{template .unusedShape}
+{let $numNotices: length($notices)/}
+  <p class="error">Error - Unused shape!</p>
+  <p>Description: Every shape in shapes.txt is used by some trip from trips.txt.</p>
```

Comment:
should be used?

Reply:
Thank you for pointing it out! I have changed the description to "The shape in shapes.txt is never used by any trip from trips.txt.".